### ISSUE TITLE
Toast of type `loading` doesn't display spinner

### ISSUE THREAD

**Issue by ImBaedin** [emilkowalski/sonner#354]
### Describe the feature / bug 📝:

When creating a loading spinner, there is a space for an icon but no icon.

### Steps to reproduce the bug 🔁:

```ts
const loadingId = toast.loading('Loading...');
...
toast.success('Done loading', {
  id: loadingId
})
```
<img width="420" alt="Screenshot 2024-02-27 at 1 04 21 AM" src="https://github.com/emilkowalski/sonner/assets/7637729/c889a0a5-de10-49e7-b82f-fb8370938c94">

**Comment by AdiRishi** [comment]
Just ran into this myself. Looks like it breaks in `1.4.2` but still works in `1.4.1`

**Comment by emilkowalski** [comment]
Fixed in #359
